source/release-notes.txt

```diff
@@ -10,6 +10,37 @@ Release Notes
    :depth: 1
    :class: twocols
 
+
+1.9.0 Changelog
+---------------
+
+*Released August 20, 2024*
+
+New features:
+
+- Query Converter now allows you to test your relational and converted 
+  queries and compare the results in the . 
```

Comment:
Incomplete sentence

Reply:
Woops forgot to update this, I was checking what we referred to the feature in the other branch. Updated.